utils/micro.py

```diff
@@ -137,11 +132,14 @@ def config_logging(debug):
     """Config logging level output output"""
 
     if debug:
-        logging.basicConfig(level=logging.DEBUG, format='%(asctime)s %(message)s')
+        logging.basicConfig(level=logging.DEBUG,
+                            format="[%(asctime)s - %(name)s - %(levelname)s] - %(message)s")
         logging.debug("Debug mode activated")
     else:
         logging.basicConfig(level=logging.INFO, format='%(asctime)s %(message)s')
 
+    logging.getLogger('elasticsearch').setLevel(logging.INFO)
```

Comment:
It would be possible to add a comment to explain why the elasticsearch logger is always set to INFO ?